ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -848,6 +848,27 @@ tests:
       enable:
       - observers-resource-watch
     workflow: network-flow-matrix-single-node-bm
+- as: commatrix-e2e-tests
```

Comment:
These should follow our naming scheme for CI jobs

https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/

Mostly I think you're missing platform and CNI provider.

If these are the same style jobs as https://github.com/openshift/release/pull/56792, why doesn't it also say "network-flow-matrix" in the name?

Reply:
Hey will change the name it will be in the same style, currently just testing it :)